apps/fishing-map-e2e/src/constants/buttons.ts

```diff
@@ -1,3 +1,14 @@
 export const SIDEBAR_TOGGLE_EEZ = 'context-layer-context-layer-eez'
 
 export const MAP_POPUP_EEZ_SECTION = 'context-tooltip-section-public-eez-areas'
+
+export const REPORT_FORMAT_JSON = 'report-format-json'
+export const REPORT_FORMAT_CSV = 'report-format-csv'
+
+export const GROUPBY_FLAG = 'group-vessels-by-flag'
+export const GROUPBY_MMSI = 'group-vessels-by-mmsi'
+export const GROUPBY_GEAR = 'group-vessels-by-gearType'
+export const GROUPBY_FLAG_GEAR = 'group-vessels-by-flagAndGearType'
+
+export const GROUPBY_DAY = 'group-time-by-daily'
+export const GROUPBY_MONTH = 'group-time-by-monthly'
```

Comment:
If these constants are exclusively used in the `eez-download` test, I'd better move them to `eez-download.po.ts` as explained [below](https://github.com/GlobalFishingWatch/frontend/pull/2154/files#diff-adc1e64389f81e8dcfdb8819c4468f695ab2ce022cfbb0c03fce50699dd6942bR71)

Reply:
I dont now if this will be used only for this test, so I think they can stay here